static/contact.html

```diff
@@ -178,6 +178,7 @@ <h3><a href="#standalone-apps">Standalone apps</a></h3>
                         <li><a href="https://github.com/GrapheneOS/Auditor/issues">Auditor issue tracker</a></li>
                         <li><a href="https://github.com/GrapheneOS/PdfViewer/issues">PDF Viewer issue tracker</a></li>
                         <li><a href="https://github.com/GrapheneOS/Vanadium/issues">Vanadium (Chromium variant) issue tracker</a></li>
+                        <li><a href="https://github.com/GrapheneOS/Camera/issues">GrapheneOS Camera app issue tracker</a></li>
```

Comment:
Second-guessing adding that "GrapheneOS" part to the Camera link.

It's implied by the other links that these are _by_ GrapheneOS. Feel free to remove it.